deploy/README.md

```diff
@@ -0,0 +1,16 @@
+Bringing up a local instance of DBSP
+===================================
+
+First, build a DBSP Docker image:
+
+```
+./docker.sh
+```
+
+Next, bring up an instance of the container:
+
+```
+docker run --name dbsp -p 8081:8080 -itd dbspmanager
```

Comment:
Is this in case we're already running the DBSP manager on the host?

Reply:
Yup. 8080 is commonly used across projects, but this is just to show you can pick the port you want to forward to.

Reply:
I've also clarified in the README that the user can pick any port to forward to.